client/document/support-user.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+function supportUserFn( { user, token, authorized } ) {
+	const url = window.location.toString();
+
+	if ( url.indexOf( '?' ) > 0 ) {
+		const cleanUrl = url.substring( 0, url.indexOf( '?' ) );
+		window.history.replaceState( {}, document.title, cleanUrl );
+	}
+
+	if ( authorized ) {
+		window.sessionStorage.setItem( 'boot_support_user', JSON.stringify( { user, token } ) );
+	}
+
+	window.location.replace( '/' );
+}
+
+function SupportUser( { supportUser, supportToken, authorized = false, urls, jsFile, config } ) {
+	return (
+		<html lang="en">
+			<body>
+				{ /* eslint-disable react/no-danger */ }
+				{ config && <script type="text/javascript">{ config }</script> }
+				<script src={ urls.manifest } />
+				<script src={ urls.vendor } />
+				<script src={ urls[ jsFile ] } />
```

Comment:
We explicitly don't want other 3rd party scripts here right?

Reply:
That's only what is needed to bootstrap the current approach for shipping the inline script. Note that it's obsolete if we don't go for the current approach.